syntax="proto3";

package SmartOffice;

service AirConditioner{
    rpc OnOffAirCond (RequestAirConditioner) returns (ResponseAirConditioner) {}
    rpc UpperTemp (RequestAirConditioner) returns (ResponseAirConditioner) {}
    rpc LowerTemp (RequestAirConditioner) returns (ResponseAirConditioner) {}
    rpc GetAirCondInfo (RequestAirConditioner) returns (ResponseAirConditioner) {}
    rpc OnOffTemperatureSensor(RequestAirConditioner) returns (ResponseAirConditioner) {}
}


message RequestAirConditioner {}

message ResponseAirConditioner{
    string tipo = 1;
    string name = 2;
    bool status = 3;
    int32 temperature = 4;
}